tracing-flame/src/lib.rs

```diff
@@ -235,6 +246,16 @@ where
             out: self.out.clone(),
         }
     }
+
+    pub fn filter_empty(mut self) -> Self {
```

Comment:
Can we get comments on these methods explaining what they do and why you'd want them?